### DISCUSSION TITLE
File access in rust

### DISCUSSION THREAD

**Discussion by tygyh** [bevyengine/bevy#10919] [Q&A] [answered]
I'm trying to add some helper functions to the examples which mean I would want to put the `helpers` file in the folder for it to be available in all the subfolders. Since that means it is not in the same module as any of the files I want to access it from, I do not know what is required for it to work.
There is also an error about there not being a `main` function in the `helper` file, but I do not know if that is related to the first issue.

**Comment by NiklasEi** [comment]
You could create a separate crate with helpers for examples and add it as a development dependency.

But I am not convinced that would be a good thing to do. It's valuable, that users can copy paste the content of an example and just run it. That would no longer work if the example uses some help methods defined outside of bevy and the example.

**Reply by tygyh** [reply, marked as answer]
Since you do not think it would be a good idea to create such a crate I do not see a reason to do so.
